src/app/ApplicationTemplate.Shared/Business/Impl/ChuckNorrisService.cs

```diff
@@ -26,7 +26,7 @@ public async Task<ChuckNorrisQuote[]> Search(CancellationToken ct, string search
 		{
 			// If the search term does not contain at least 3 characters, the API returns an exception.
 			// It must be handle on app side.
-			if (searchTerm.Length < 3)
+			if (string.IsNullOrEmpty(searchTerm) || searchTerm.Length < 3)
```

Comment:
you can probably write simply do `searchTerm?.Length < 3`

Reply:
doing like you said is still causing an error. Because it is null, it will count as false and skip the if.